dbt/adapters/athena/__version__.py

```diff
@@ -0,0 +1 @@
+version = "1.3.2"
```

Comment:
💡 If we want to automate some things like bumping the version in `__version__.py` and `setup.py` in the future: i've used https://github.com/release-it/release-it in the past which has a plugin https://github.com/release-it/bumper

```
"plugins": {
  "@release-it/bumper": {
    "out": ["setup.py", "dbt/adapters/athena/__version__.py"]
  }
}
```

Then, when we run the command locally or in CI, it will generate the next version number (patch/minor/major), bump the version in all the files specified, generate a changelog, create a release and tag, ...

Reply:
I'm wonder how such tool works when we need to bump minor version (I've seen that dbt is already pushing for version 1.4). Indeed, I'm with you on making the bumping process as smooth as possible.